packages/api/tests/lib/rules/test-updateRuleResources.js

```diff
@@ -111,146 +111,6 @@ test.after.always(async (t) => {
   });
 });
 
-test.serial('Updating rule triggers for a kinesis type rule updates event mappings', async (t) => {
```

Comment:
These assertions are no longer correct according to the changes in main.

As an aside, I've tried to not change assertions as much as possible but the rules tests are spread out among several files and are very difficult to parse and ensure we have complete coverage. A separate ticket might be necessary (and difficult).

Reply:
Separate ticket: https://bugs.earthdata.nasa.gov/browse/CUMULUS-3204